bundle/manifests/storageos-operator-leader-election_rbac.authorization.k8s.io_v1_rolebinding.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  labels:
+    app: storageos
+    app.kubernetes.io/component: operator
+  name: storageos-operator-leader-election
```

Comment:
I think this should be: `storageos:leader-election" or even `storageos:leader-locking-operator` to match:

```
NAMESPACE     NAME                                                ROLE                                                  AGE
kube-public   kubeadm:bootstrap-signer-clusterinfo                Role/kubeadm:bootstrap-signer-clusterinfo             17m
kube-public   system:controller:bootstrap-signer                  Role/system:controller:bootstrap-signer               17m
kube-system   kube-proxy                                          Role/kube-proxy                                       17m
kube-system   kubeadm:kubelet-config-1.20                         Role/kubeadm:kubelet-config-1.20                      17m
kube-system   kubeadm:nodes-kubeadm-config                        Role/kubeadm:nodes-kubeadm-config                     17m
kube-system   system::extension-apiserver-authentication-reader   Role/extension-apiserver-authentication-reader        17m
kube-system   system::leader-locking-kube-controller-manager      Role/system::leader-locking-kube-controller-manager   17m
kube-system   system::leader-locking-kube-scheduler               Role/system::leader-locking-kube-scheduler            17m
kube-system   system:controller:bootstrap-signer                  Role/system:controller:bootstrap-signer               17m
kube-system   system:controller:cloud-provider                    Role/system:controller:cloud-provider                 17m
kube-system   system:controller:token-cleaner                     Role/system:controller:token-cleaner                  17m
kube-system   weave-net                                           Role/weave-net                                        17m
storageos     storageos-leader-election-rolebinding               Role/storageos-leader-election-role                   14m
```